chocolatey/lektor.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>lektor</id>
+    <version>3.2.0</version>
+    <packageSourceUrl>https://github.com/lektor/lektor</packageSourceUrl>
+    <owners>Nimish Bongale</owners>
```

Comment:
I'm happy with you taking credit in some way (like the description), but I think the `owners` field should be something like _The Lektor Team_.

Reply:
Yep! This is again autogenerated by chocolatey. Since I created the package on the site, it by default adds me as the owner. I'm happy to change it as per your direction.